vortex-datafusion/src/convert/exprs.rs

```diff
@@ -267,6 +272,7 @@ fn supported_data_types(dt: &DataType) -> bool {
                 | Timestamp(_, _)
                 | Time32(_)
                 | Time64(_)
+                | Struct(_)
```

Comment:
I think we should probably expand this list further? I just added Struct to make one of the tests pass